lib/syntax_tools/src/erl_syntax_lib.erl

```diff
@@ -23,9 +23,6 @@
 %%
 %% This module contains utility functions for working with the
 %% abstract data type defined in the module {@link erl_syntax}.
-%%
-%% @type syntaxTree() = erl_syntax:syntaxTree(). An abstract syntax
```

Comment:
Please keep this local type. It will make the documentation more readable. E.g. it will generate

<code>Function = (syntaxTree(), term()) -> {syntaxTree(), term()}</code>

instead of

<code>Function = (syntaxTree() (see module erl_syntax), term()) -> {syntaxTree() (see module erl_syntax), term()}</code>